extract_pipeline.py

```diff
@@ -0,0 +1,41 @@
+import requests
+import pandas as pd
+
+response = requests.get('https://randomuser.me/api/')
+
+data = response.json()
+results = data['results']
+
+normalized_result = pd.json_normalize(results)
+
+#DATA TRANSFORMATION
+#Convert to dataframe
+
+normalized_frame = pd.DataFrame(normalized_result)
+
+
+##DATA TRANSFORMATION: Select Columns
+
+selected_columns = normalized_frame[['name.title', 'name.first', 'name.last', 'cell','email','location.street.number','location.street.name','location.city','location.country',
+                  'login.username', 'login.password', 'dob.date', 'dob.age']]
+
+
+#DATA TRANSFORMATION: Rename Columns
+
+renamed_normalize = selected_columns.rename(columns={
```

Comment:
You can transform the column from the initial normalised_result variable

Reply:
Do you mean I can make a subset of the columns or that I can simply rename them?

Reply:
rename the columns , please create a module that rename columns in a pandas dataframe , your colleague can reuse such module in the future

Reply:
On it , sir